blog/_posts/btcpay-server-2-0.md

```diff
@@ -0,0 +1,174 @@
+
+[![](https://img.youtube.com/vi/DnASnEdegn4/mqdefault.jpg)](https://www.youtube.com/watch?v=DnASnEdegn4)
+
+Get started by downloading the [plugin on GitHub](https://github.com/btcpayserver/odoo) and following our [Odoo documentation](https://docs.btcpayserver.org/Odoo/).
+
+### BigCommerce 
+
+BigCommerce is a leading e-commerce platform designed for scalability and focused on enterprise-level merchants. We’re bringing Bitcoin payments to BigCommerce users with our latest integration. 
+
+[![](https://img.youtube.com/vi/lSsk80P91Jo/mqdefault.jpg)](https://www.youtube.com/watch?v=lSsk80P91Jo)
+
+To start accepting Bitcoin for your BigCommerce store, check our [documentation](https://docs.btcpayserver.org/BigCommerce/). 
+
+### Other integration updates
+
+We’ve done extensive tests for all of our ecommerce integrations. Kudos to our integration expert **Ndeet** for conducting comprehensive hard field work, [testing](https://docs.google.com/spreadsheets/d/1nfrZhchAm7AIk1dTvSSnFvnHdJ8con8XvJQqQ29khA4/edit?gid=0#gid=0) each and every one of them. 
+
+**All of our official e-commerce integrations are 2.0 compatible.**
+
+However, if you face issues with 2.0 and any official e-commerce integration, please [report it here](https://github.com/btcpayserver/btcpayserver/discussions/6300), and we’ll prioritize those reports. We continue to monitor feedback from the community and ensure existing integrations stay up-to-date. 
+
+If you're using our **Shopify** plugin, rest assured that it’s not impacted by recent API changes, and you can continue to use it without issues. We’re aware of Shopify discounting certain API calls at the end of this year, and we’re already working on shipping a brand new plugin before that happens.
+
+Additionally, our **PrestaShop** plugin received a [new update](https://github.com/btcpayserver/prestashop-plugin/releases/), thanks to our contributor BitcoinMitchell, and the PHP Library has been updated to be [compatible with BTCPay 2.0](https://github.com/btcpayserver/btcpayserver-greenfield-php/releases) as well. 
+
+## 🔌 Plugins
+
+If you use plugins with your BTCPay Server, you might notice that some are disabled after updates - we do that to prevent server crashes. If you rely heavily on plugins for your BTCPay operations, we recommend holding off on updating to version 2.0 for now. Keep an eye on announcements from developers of the plugins and give them a few more weeks to address compatibility issues and release updates.
```

Comment:
@NicolasDorier @Kukks @rockstardev should we still communicate this part?

> If you use plugins with your BTCPay Server, you might notice that some are disabled after updates - we do that to prevent server crashes. If you rely heavily on plugins for your BTCPay operations, we recommend holding off on updating to version 2.0 for now. Keep an eye on announcements from developers of the plugins and give them a few more weeks to address compatibility issues and release updates.

Now most plugins are 2.0 compatible?

Reply:
```suggestion
If you use plugins with your BTCPay Server, you may notice that some are disabled following updates - we do this to prevent server crashes. If you rely heavily on plugins for your BTCPay operations, we recommend that you double-check compatibility and look for announcements from plugin developers regarding their support for version 2.0 before proceeding with the upgrade.
```